src/main/resources/version-info.properties

```diff
@@ -0,0 +1 @@
+synapseawsconsolelogin.version = ${project.version}
```

Comment:
what is setting `project.version`?  if it's maven then don't we need to setup filtering or something?